lib/rules/time-no-imperceptible/README.md

```diff
@@ -1,5 +1,7 @@
 # time-no-imperceptible
 
+***Deprecated: this rule is outside the scope of stylelint's functionality. See [the release planning docs](https://stylelint.io/user-guide/release-planning/) for details.***
```

Comment:
We won't be releasing the `deprecations` branch until the alternative _within stylelint_ are done. So, updating this message to reference `time-min-milliseconds` SGTM. Otherwise we'll likely forget to do it when `time-min-milliseconds` is added.